runtime/Cargo.toml

```diff
@@ -24,6 +24,8 @@ smallvec = "1.6.1"
 
 # Spacewalk Dependencies
 pallet-spacewalk = {git = "https://github.com/pendulum-chain/spacewalk", default-features = false}
+pallet-pendulum-amm = {git = "https://github.com/pendulum-chain/pendulum-amm.git", package = "pallet-pendulum-amm", branch = "31-create-a-branch-and-plug-the-amm-pallet-into-pendulum", default-features = false }
```

Comment:
move to branch master after merging https://github.com/pendulum-chain/pendulum-amm/pull/21